starboard/build/config/BUILDCONFIG.gn

```diff
@@ -99,7 +99,13 @@ if (target_cpu == "x86" || target_cpu == "arm") {
 host_toolchain = "//starboard/build/toolchain/$host_os:$_host_toolchain_cpu"
 
 if (build_with_separate_cobalt_toolchain) {
-  cobalt_toolchain = "//starboard/build/toolchain:clang"
+  # TODO(b/294450490): decide a way to set cobalt_toolchain for modular builds decide for all platforms.
+  # we'll need more conditionals for other platforms.
+  if (host_os == "win") {
+    cobalt_toolchain = "//$starboard_path/toolchain:cobalt"
```

Comment:
Is this toolchain already defined? I can't find it.

Reply:
This will be defined for platforms like ps4 in a follow up change : https://lbshell-internal-review.git.corp.google.com/c/cobalt_src/+/249080/20